pkg/ccl/crosscluster/physical/stream_ingestion_dist.go

```diff
@@ -515,12 +539,18 @@ func (p *replicationFlowPlanner) constructPlanGenerator(
 		if err != nil {
 			return nil, nil, err
 		}
+
+		// If we have fewer partitions than we have nodes, try to repartition the
+		// topology to have more partitions.
+		topology = repartitionTopology(topology, len(sqlInstanceIDs)*8)
```

Comment:
any reason you wanted to remove this tunable cluster setting? Like, if we encounter another pcr oom, it may be nice to have it.

Reply:
The OOM was in smaller dest clusters than src clusters where we'd create >8 procs per node, whereas now we are using the src cluster size directly here, so we no longer risk that specific behavior. The setting also wasn't settable in CC where the sys tenant is inaccessible so it wasn't going to be useful anyway.